src/mon/Monitor.cc

```diff
@@ -1433,12 +1433,13 @@ void Monitor::handle_sync_get_chunk(MonOpRequestRef op)
   while (sp.last_committed < paxos->get_version() && left > 0) {
     bufferlist bl;
     sp.last_committed++;
-    store->get(paxos->get_name(), sp.last_committed, bl);
-    // TODO: what if store->get returns error or empty bl?
+
+    int err = store->get(paxos->get_name(), sp.last_committed, bl);
+    assert (err == 0 && bl.length() > 0);
+
     tx->put(paxos->get_name(), sp.last_committed, bl);
     left -= bl.length();
-    dout(20) << __func__ << " including paxos state " << sp.last_committed
-	     << dendl;
+    dout(20) << __func__ << " including paxos state " << sp.last_committed << dendl;
```

Comment:
I may be wrong, as I only checked with copy-pasting it and running the line through python's `len()`, but this line now has 80+ columns. Please make sure it's wrapped at 80.